docs/source/notebooks/wps_CI_demo.ipynb

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
<p>Is the <code>while</code>  loop necessary for this demo to work?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/chickadee/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I believe this is included in every bird we have. It was the best way Eric and I could find to set the top <code>chickadee</code> directory as the working directory</p>